### PR TITLE
HW 55-22

### PR DESCRIPTION
**Pull request by vitalka1994**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,56 @@
...
+    private int speed;
+    private int weight;
+    private String brand;
+
+    private Plane(PlaneBuilder builder) {
+        this.model = builder.model;
+        this.color = builder.color;
+        this.speed = builder.speed;
+        this.weight = builder.weight;
+        this.brand = builder.brand;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: do we need to use `this` keyword here?

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,56 @@
...
+
+        public PlaneBuilder setBrand(String brand) {
+            this.brand = brand;
+            return this;
+        }
+
+        public Plane build() {
+            return new Plane(this);
+        }
+    }
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: please remove this blank line

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,5 +7,55 @@
...
+
+    public static class PlaneBuilder {
+        private String model;
+        private String color;
+        private int speed;
+        private int weight;
+        private String brand;
+
+        public PlaneBuilder setModel() {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: what are you setting here as well as in all other setters? you do not pass anything to methods

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Plane.java [hunk 4]
```diff
@@ -7,5 +7,55 @@
...
+            this.brand = brand;
+            return this;
+        }
+
+        public Plane build() {
+            return new Plane(this);
+        }
+    }
 }
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java:61]:
```suggestion
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**